packages/controllers/src/controllers/ConnectorController.ts

```diff
@@ -327,6 +329,16 @@ const controller = {
     return ConnectorController.mergeMultiChainConnectors(namespaceConnectors)
   },
 
+  canSwitchToSmartAccount(namespace: ChainNamespace) {
```

Comment:
If a method doesn't mutate the controller state, I think it should be a util rather then living inside the controller file